src/components/Dashboard/TalkDashboard.vue

```diff
@@ -351,21 +367,37 @@ function scrollEventCards({ direction }: { direction: 'backward' | 'forward' })
 @import '../../assets/variables';
 
 .talk-dashboard-wrapper {
-	--title-height: calc(var(--default-clickable-area) + var(--default-grid-baseline) * 3); // '.title' height
-	--section-width: 300px;
-	--section-height: 300px;
-	--content-height: calc(100% - var(--title-height));
-	padding: 0 calc(var(--default-grid-baseline) * 3);
-	max-width: calc($messages-list-max-width + 400px); // FIXME: to change to a readable value
+	padding: calc(var(--default-grid-baseline) * 2) calc(var(--default-grid-baseline) * 3);
+	width: calc(100vw - 300px - var(--body-container-margin) * 2); // 300px for the left sidebar and body container margins
 	margin: 0 auto;
+	display: flex;
+	flex-direction: column;
+	height: 100%;
+	max-height: 800px;
+	max-width: 1200px;
+
+	&--mobile {
+		width: 100%;
+		grid-template-columns: 1fr;
+
+		.talk-dashboard__chats {
+			grid-template-columns: 1fr;
+			gap: 0;
+		}
+
+		.event-section {
+			width: 350px;
```

Comment:
I'd remove this style, looks better without limitation on both filled and empty section
a | b
-- | --
<img width="437" height="231" alt="Image" src="https://github.com/user-attachments/assets/b61cbed6-ce05-4944-81c9-e1d2d3212e7f" /> | <img width="384" height="231" alt="Image" src="https://github.com/user-attachments/assets/7a5ddb94-d616-4565-bb16-d9226e2ef619" />